WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -56,6 +56,21 @@ def update_events_from_wcif
     }
   end
 
+  def update_persons_from_wcif
```

Comment:
I had to implement such an endpoint for the schedule too, and ended up DRYing up the code for the exception handling, see [here](https://github.com/viroulep/worldcubeassociation.org/commit/8aa06b15d9b58a838412a40ae66642da2add9595).

I'm still not done with the test unfortunately, so your PR is likely to be ready faster than mine...
If you think it's a good idea maybe you could take this change right away?

Isn't there a JSON schema validation for the persons WCIF?

Reply:
I'm totally with fixing the conflict myself so if you can just include the changes made to the API competitions controller (and use it of course :p) that's awesome (L39-61 [here](https://github.com/viroulep/worldcubeassociation.org/blob/8aa06b15d9b58a838412a40ae66642da2add9595/WcaOnRails/app/controllers/api/v0/competitions_controller.rb#L39-L61))!
This way I'll just drop the changes to the file and add the `update_schedule_from_wcif` endpoint on top of it.

Reply:
Sounds good, thanks!